I/ITK/build_tarballs.jl

```diff
@@ -0,0 +1,116 @@
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ITK"
+version = v"5.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/InsightSoftwareConsortium/ITK.git", "1fc47c7bec4ee133318c1892b7b745763a17d411")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ITK*
+mkdir build/
+cmake -B build -S . \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DBUILD_SHARED_LIBS:BOOL=ON \
+    -DITK_USE_SYSTEM_EXPAT:BOOL=ON \
+    -DITK_USE_SYSTEM_FFTW:BOOL=ON \
+    -DITK_USE_SYSTEM_HDF5:BOOL=ON \
+    -DITK_USE_SYSTEM_JPEG:BOOL=ON \
+    -DITK_USE_SYSTEM_TIFF:BOOL=ON \
+    -DITK_USE_SYSTEM_PNG:BOOL=ON \
+    -DITK_USE_SYSTEM_EIGEN:BOOL=ON \
+    -DITK_USE_SYSTEM_ZLIB:BOOL=ON \
+    -DQNANHIBIT_VALUE:BOOL=0 \
+    -DQNANHIBIT_VALUE__TRYRUN_OUTPUT:STRING=0 \
+    -DVXL_HAS_SSE2_HARDWARE_SUPPORT:STRING=1 \
+    -DVCL_HAS_LFS:STRING=1 \
+    -DDOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS:STRING=1 \
+    -DHAVE_CLOCK_GETTIME_RUN:STRING=0
+cmake --build build --parallel ${nproc}
+cmake --install build
+install_license ${WORKSPACE}/srcdir/ITK/LICENSE
+"""
+
+
+
+#sse2 disabled errors in ITK with open issues on github for i686 platforms
+#CMAKE errors for _libcxx_run_result in cross compilation for macOS, freebsd and x86_64 linux musl
+#
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = platforms = BinaryBuilder.supported_platforms(exclude=x -> (Sys.isapple(x) || Sys.isfreebsd(x) || arch(x) == "i686" || (arch(x) == "x86_64" && libc(x) == "musl")))
+platforms = expand_cxxstring_abis(platforms)
```

Comment:
Based on my experience, I really would suggest structuring it like this, i.e.: group the comment with the resulting action; and keep each removal on its own line so it is easier to experiment with it in the future
```suggestion
# These are the platforms we will build for by default, unless further
# platforms are passed in on the command line
platforms = supported_platforms()

#sse2 disabled errors in ITK with open issues on github for i686 platforms
filter!(p -> arch(p) == "i686", platforms)

#CMAKE errors for _libcxx_run_result in cross compilation for macOS, freebsd and x86_64 linux musl
filter!(!Sys.isapple, platforms)
filter!(!Sys.isfreebsd, platforms)
filter!(p -> !(arch(x) == "x86_64" && libc(x) == "musl"), platforms)

platforms = expand_cxxstring_abis(platforms)
```